.github/workflows/build-tauri.yml

```diff
@@ -1,5 +1,20 @@
-name: "Build And Publish Desktop Tauri Apps"
-on: [workflow_dispatch]
+name: Build And Publish Desktop Tauri Apps
+
+on:
+    workflow_dispatch:
+        inputs:
+            environment:
+                type: environment
+
+    push:
+        branches:
+            - "v4"
+            - "master"
+            - "feature/**"
+            - "fix/**"
+        paths:
+            - "packages/electron/**"
+            - "assets/**"
```

Comment:
@MaKleSoft I actually am not sure if these are enough (or if there's anything else.)

Reply:
The Tauri version actually builds and bundles the app code instead of loading the pwa remotely, se we do need the other packages (app, core, locale). Also, change `packages/electron` to `packages/tauri` obviously.